docs/src/ScrollableContainer.doc.js

```diff
@@ -391,195 +400,195 @@ function ScrollableContainerExample() {
-                    paddingX={3}
-                    paddingY={3}
+                      <Dropdown.Item
+                        handleSelect={handleSelect}
+                        selected={selected}
+                        option={{
+                          value: "Headquarters San Francisco",
+                          label: "Headquarters San Francisco",
+                          subtext: "516 Natoma Street, Suite # 23" }}
+                      />
+                      <Dropdown.Item
+                        handleSelect={handleSelect}
+                        selected={selected}
+                        option={{
+                          value: "Headquarters Seattle",
+                          label: "Headquarters Seattle",
+                          subtext: "123 Main Street, Suite # 48" }}
+                      />
+                    </Dropdown>
+                  )}
+                </Box>
+                <Box
+                  flex="grow"
+                  paddingX={3}
+                  paddingY={3}
+                >
+                  <Heading
+                    size="sm"
+                    accessibilityLevel={2}
                   >
-                    <TextField
-                      label="Address Line 2"
-                      id="Address_Line_2"
-                      onChange={() => {}}
-                    />
-                  </Box>
+                    Billing Address
+                  </Heading>
+                </Box>
+                <Box
+                  flex="grow"
+                  paddingX={3}
+                  paddingY={3}
+                >
+                  <TextField
+                    label="Address Name"
+                    id="Address_Name"
+                    onChange={() => {}}
+                  />
+                </Box>
+                <Box
+                  flex="grow"
+                  paddingX={3}
+                  paddingY={3}
+                >
+                  <TextField
+                    label="Business Name"
+                    id="Business_Name"
+                    onChange={() => {}}
+                  />
+                </Box>
+                <Box
+                  flex="grow"
+                  paddingX={3}
+                  paddingY={3}
+                >
+                  <TextField
+                    label="Address Line 1"
+                    id="Address_Line_1"
+                    onChange={() => {}}
+                  />
+                </Box>
+                <Box
+                  flex="grow"
+                  paddingX={3}
+                  paddingY={3}
+                >
+                  <TextField
+                    label="Address Line 2"
+                    id="Address_Line_2"
+                    onChange={() => {}}
+                  />
+                </Box>
+                <Box
+                  flex="grow"
+                  paddingX={3}
+                  paddingY={3}
+                >
                   <Box
```

Comment:
ditto here on using Flex to simplify the markup on this parent and all the children

Reply:
Let's update this in a separate PR, along with Layouts.